packages/api-gateway/src/drivers/http/adapters/admin_panel/handlers.js

```diff
@@ -1,11 +1,35 @@
 async function sayHello(req, reply) {
-  const result = await this.adminPanelService.sayHello();
+
+	const result = await this.adminPanelService.sayHello();
 
   return reply.code(200)
     .header('Content-Type', 'application/json; chartset:utf-8')
     .send({ result });
 }
 
+async function changeUserStatus(req, reply) {
+
+  //TODO: this is not the place for try/catch
```

Comment:
Can add an space after "//"? is throwing eslint warnings
```suggestion
  // TODO: this is not the place for try/catch
```